miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditBoxController.java

```diff
@@ -176,4 +197,37 @@ public String processSubmit(@ModelAttribute("box") Box box, ModelMap model, Sess
     model.clear();
     return "redirect:/miso/box/" + box.getId();
   }
+
+  private final class BulkCreateBoxBackend extends BulkCreateTableBackend<BoxDto> {
+    public BulkCreateBoxBackend(Class<? extends BoxDto> dtoClass, BoxDto dto, Integer quantity) {
```

Comment:
In these constructors, it's preferred to make them only take the quantity argument and fill in the DTO and class with literals in the call to `super`.